roofit/roofitcore/src/RooAddModel.cxx

```diff
@@ -105,7 +105,7 @@ RooAddModel::RooAddModel(const char *name, const char *title, const RooArgList&
     msgSs << "RooAddModel::RooAddModel(" << GetName()
           << ") number of pdfs and coefficients inconsistent, must have Npdf=Ncoef or Npdf=Ncoef+1";
     const std::string msgStr = msgSs.str();
-    coutE(InputArguments) << msgStr << endl;
+    coutE(InputArguments) << msgStr << "\n";
```

Comment:
I wonder if all systems flush the stream (i.e. print to the terminal or the respective message handler) before the exception arrives?
Maybe it's a good idea to let it flush?

Reply:
I believe this is already handled [here](https://github.com/root-project/root/blob/5aee32a4c137a6ed31ee199ee8c80a6291f2d2e6/roofit/roofitcore/src/RooMsgService.cxx#L500-L506)
and in reality I don't think even `std::endl` can guarantee this. This is OS-dependent and possibly only can be done with [`std::set_terminate`](https://en.cppreference.com/w/cpp/error/set_terminate).

Reply:
I don't think it's important, so no need to insist. 👍